### DISCUSSION TITLE
Tasmota LVGL and touch

### DISCUSSION THREAD

**Discussion by vid8266** [arendst/Tasmota#18097] [Q&A]
Good afternoon! Help solve the problem using the ILI9488 display and Tasmota LVGL(ESP32). Loaded uDisplay, LVGL example works, DisplayValibrate works fine too. But in the example, touch does not respond to the described buttons prev, next, home. What's wrong? An attempt to prescribe lv_obj_set_event_cb from the LVGL examples (instead of the prev_btn.add_event_cb(btn_clicked_cb, lv.EVENT_CLICKED, 0) suggested in the example) produces an error.
The specified event - prev_btn.add_event_cb(btn_clicked_cb, lv.EVENT_CLICKED, 0) should respond to touch? As far as I understand from the DisplayValibrate application, the driver for touch is loaded there. How to work with screen objects using touch in tasmota LGVL?
Sorry if I didn't describe everything correctly. I use Tasmota as a very good control tool and don't want to leave it for display work.
Guru, help me understand how to work with this!!

**Comment by s-hadinger** [comment]
I'm out right now. I will be able to help early next week

**Comment by vid8266** [comment]
Thank you! It's just that I've been messing around with this for a long time and I just can't get it. Not in a hurry, I will be grateful for the help !!!

**Comment by vid8266** [comment]
Adventures with LVGL + Berry + Tasmota #11784 - I read this discussion, but still have questions.
